include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -17,11 +20,11 @@ namespace shapes
     // GH link https://github.com/erich666/GraphicsGems/blob/master/gems/Roots3And4.c
     // Credits to Doublefresh for hinting there
     // returns the roots, and number of filled in real values under numRealValues
-    float2 SolveQuadratic(float3 c, out int numRealValues)
+    float2 SolveQuadratic(float3 C, out int numRealValues)
```

Comment:
Remove this function, Lucas has worked on a new version that doesn't do this.

Reply:
done